### ISSUE TITLE
Some strange about this part?

### ISSUE THREAD

**Issue by zhiweihu1103** [dyang67/GammaE#1]
Hi Dong, I dont quite understand this part code, I see KGReasoning dont have the code from 272 to 277. 
https://github.com/dyang67/GammaE/blob/fd34b8de93ef2be2e0649ac179d1bf49cd81c168/main.py#L272

Because if this piece of code is included, the following error will occur during the verification phase:
```python
File "../main.py", line 446, in main
    logging.info('Evaluating on Valid Dataset...')
  File "../main.py", line 158, in evaluate
    metrics = model.test_step(model, tp_answers, fn_answers, args, dataloader, query_name_dict)
  File "/work/zhiweihu/reproduce-code/new_code/Quaternion-CQA/GammaE-CQA/models.py", line 495, in test_step
    batch_queries_dict[query_structure] = torch.LongTensor(batch_queries_dict[query_structure])
AssertionError

**Comment by Vin1115** [comment]
2u and up are zero-shot queries, and the so-called union network is trained by transferring union queries from the validation set to the training set. This is unfair compared to other methods that have never been trained on union queries.
